src/components/AutoSearch.tsx

```diff
@@ -50,9 +54,9 @@ export const AutoSearch: FC<AutoSearchProps> = ({
 
   useEffect(() => {
     if (!value) {
-      setDisplayValue({ title: '' });
+      setDisplayValue(emptyValue);
     }
-    setDisplayValue({ title: value });
+    setDisplayValue({ title: value || '' });
```

Comment:
Bruk emptyValue her også

Reply:
Hvis det er en value der skal den brukes, så tom streng er bare hvis value er `null` eller `undefined`. Men kan legge tom streng som default for propen, blir kanskje litt ryddigere